client/src/components/Sub_ProductDetail/Sub_ProductInfo/SocialMediaShare.jsx

```diff
@@ -0,0 +1,28 @@
+import React, { useState } from 'react';
+import { AiFillTwitterCircle } from 'react-icons/ai';
```

Comment:
I think these could go in 1 line?

Reply:
not sure how to do that. because line 1 and line 2 are imported from different packages. line 1 is 'react' and line 2 is 'react-icons/ai'